src/hotspot/share/oops/constantPool.cpp

```diff
@@ -298,20 +298,22 @@ objArrayOop ConstantPool::prepare_resolved_references_for_archiving() {
 
   objArrayOop rr = resolved_references();
   if (rr != nullptr) {
-    ConstantPool* orig_pool = ArchiveBuilder::current()->get_source_addr(this);
```

Comment:
Are the changes below necessary? I think the original was fine but I may be missing the point of this change.

Reply:
It's just for consistency. "source" is the terminology used in the comments in archiveBuilder.cpp.